### ISSUE TITLE
Scrollbar has top padding

### ISSUE THREAD

**Issue by Saxorman** [salim-lachdhaf/searchable_dropdown#512]
**Describe the bug**
The scrollbar has an indent at the top. Adding properties did not change the situation. Checked on the following versions of the library: 4.0.0, 5.0.0, 5.0.3 (others have not been checked). The problem is observed in my project and in the example.
This problem is observed when creating menu, dialog

**To Reproduce**
Open list and scroll

**Expected behavior**
The default scrollbar will not have this padding and will start at the beginning of the list

**Screenshots**
![dropdownsearch](https://user-images.githubusercontent.com/14909862/203964227-c3f8f6be-e457-4433-9628-51ef62161352.gif)

**Smartphone:**
 - Device: emulator of android studio, Xiaomi Mi8
 - OS: Android 10, Android 11 (other not tested)

**Additional context**
Flutter 3.3.8
Dart 2.18.4

**Comment by salim-lachdhaf** [comment]
thanks for reporting that, i will check it.

**Comment by rorystephenson** [comment]
This is caused by SafeArea. My understanding is that the popup already takes in to account the SafeArea when positioning the popup which means it has consumed the view insets and should remove them from the BuildContext provided to the popup widget.

I am no longer using this library as I have implemented my own but I noticed the scroll padding issue before and also noticed it in my solution so I wanted to share the fix. In my solution it looks like:

```dart
    return navigator
        .push(
      _PopupMenuRoute<T>(
        context: context,
        position: position,
        child: MediaQuery(
          // Remove the view insets from MediaQuery as they are already
          // accounted for. If they are not removed widgets like ScrollBar will
          // have unnecessary padding.
          data: MediaQuery.of(context).copyWith(viewInsets: EdgeInsets.zero),
          child: widget.popupBuilder(context),
        ),
        capturedThemes: InheritedTheme.capture(
          from: context,
          to: navigator.context,
        ),
      ),
    );
```

**Comment by salim-lachdhaf** [comment]
fixed in the next version v6.0.0-pre that will be released in next days
